### PR TITLE
added hacking DC to ASPRO-S

### PR DESCRIPTION
**Pull request by bleehu**
Low priority; I don't think anyone uses an ASPRO anyway.

### CODE REVIEW

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 1]
```diff
@@ -981,7 +981,16 @@ explosive ordinance. When active, this system destroys any incoming grenades,
...
-hostile Nanite attacks, see GM for specifics.
+hostile Nanite attacks, see GM for specifics. 
+
+    The sensors that detect incoming explosive ordinance are rather sensitive
+and can easily be disrupted with a hacking DC check of 45 as long as the hacker
+has an access point within 20m. If a hacker surpasses a DC of 120, the ASPRO-S
+can be pwned and all of the defusing charges can be used to make a single 
+ranged attack roll up to a range of 6m with a 4 chance to miss. If the attack
+hits, it deals 80 damage with an armor piercing level of 3. Firing the attack
+depletes the ASPRO-S, which doesn't have any more charges to diffuse incoming
```
Comment (Turtlelord26) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: "diffuse" -> defuse, except I think you want "intercept"

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 2]
```diff
@@ -981,7 +981,16 @@ explosive ordinance. When active, this system destroys any incoming grenades,
 missiles, rockets or thrown weapons that it has line of sight to from the top of 
 its 3m mast within a 6m radius. Any incoming munitions are destroyed before they 
 can detonate. Cannot stop incoming bullets, lasers, or plasma. Will stop some 
-hostile Nanite attacks, see GM for specifics.
+hostile Nanite attacks, see GM for specifics. 
+
+    The sensors that detect incoming explosive ordinance are rather sensitive
+and can easily be disrupted with a hacking DC check of 45 as long as the hacker
+has an access point within 20m. If a hacker surpasses a DC of 120, the ASPRO-S
+can be pwned and all of the defusing charges can be used to make a single
```
Comment (Turtlelord26) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: Questionable use of defuse, propose interception charges or just "all of its ammunition"

File: CharacterCreation/Class-Specific Documentation/Engineer Processes.txt [hunk 3]
```diff
@@ -981,7 +981,16 @@ explosive ordinance. When active, this system destroys any incoming grenades,
...
 can detonate. Cannot stop incoming bullets, lasers, or plasma. Will stop some 
-hostile Nanite attacks, see GM for specifics.
+hostile Nanite attacks, see GM for specifics. 
+
+    The sensors that detect incoming explosive ordinance are rather sensitive
+and can easily be disrupted with a hacking DC check of 45 as long as the hacker
+has an access point within 20m. If a hacker surpasses a DC of 120, the ASPRO-S
+can be pwned and all of the defusing charges can be used to make a single 
+ranged attack roll up to a range of 6m with a 4 chance to miss. If the attack
+hits, it deals 80 damage with an armor piercing level of 3. Firing the attack
```
Comment (Turtlelord26) [inline CharacterCreation/Class-Specific Documentation/Engineer Processes.txt]: AP level 3 seems high, the aspro doesn't need to hit ordnance all that hard to nullify it.

**Review by Turtlelord26 — APPROVED** [review]
Seems like a reasonable handling to send to test branch. As this is low priority, please handle the probably-not-blocking notes inline before merging.

**Comment by bleehu** [comment]
This took care of issue #377

**Pull request merged**